src/libs/actions/Report.js

```diff
@@ -425,6 +417,65 @@ function removeOptimisticActions(reportID) {
     });
 }
 
+/**
+ * If an iouReport is open (has an IOU, but is not yet paid) then we maintain a link between it and the associated
+ * chatReport in Onyx, simplifying IOU data retrieval and reducing necessary API calls when displaying IOU components:
+ * - chatReport: {id: 123, iouReportID: 987, ...}
+ * - iouReport: {id: 987, chatReportID: 123, ...}
+ *
+ * This function allows us to fetch an iouReport without updating the linked chatReport, preventing the chatReport's
+ * 'iouReportID' value from being updated. As an example, this is desired when fetching historical reports, to avoid
+ * overwritting an open iouReportID with a closed iouReportID. If this was to occur, unpaid IOUs would not be
+ * highlighted to the user.
+ *
+ * If updating the chatReport's data is desired, use `fetchIOUReportByIDAndUpdateChatReportLink` instead.
+ *
+ * @param {Number} iouReportID - ID of the report we are fetching
+ * @param {Number} chatReportID - associated chatReportID, set as an iouReport field, but not used to maintain the link
+ */
+function fetchIOUReportByID(iouReportID, chatReportID) {
+    fetchIOUReport(iouReportID, chatReportID)
+        .then(iouReportObject => setLocalIOUReportData(iouReportObject));
+}
+
+/**
+ * If an iouReport is open (has an IOU, but is not yet paid) then we maintain a link between it and the associated
+ * chatReport in Onyx, simplifying IOU data retrieval and reducing necessary API calls when displaying IOU components:
+ * - chatReport: {id: 123, iouReportID: 987, ...}
+ * - iouReport: {id: 987, chatReportID: 123, ...}
+ *
+ * This link must remain in sync when the iouReport is modified. This function forces a link update after fetching the
+ * iouReport and therefore should only be called if we are certain that the fetched iouReport is currently open or about
+ * to be made open - else we would overwrite the existing open iouReport link with a closed iouReport.
```

Comment:
NAB 

>else we would overwrite the existing open iouReport link with a closed iouReport

Aren't we really just saying here that a `chatReport` will be updated with a new `iouReportID`? The use of `iouReport` makes this comment hard to understand.

Is the `iouReport` the **chat** that is associated with an iou report or an `iouReport`? What does it mean for an `iouReport` be "closed" vs. "open"? Can we please define what those two states mean before using them?

Reply:
> The use of iouReport makes this comment hard to understand.

Replaced iouReport with iouReportID

> What does it mean for an iouReport be "closed" vs. "open"?

Added the open iouReport explanation to this comment too.